api/types.proto

```diff
@@ -447,13 +447,25 @@ message AcceptancePolicy {
 }
 
 
+message ExternalCA {
+	enum CAType {
+		CFSSL = 0 [(gogoproto.enumvalue_customname) = "CATypeCFSSL"];
+	}
+
+	// CAType is the type of this external CA.
+	CAType type = 1;
```

Comment:
Yes. Not sure what the best wording is. I could rename it `protocol` and change the CLI parsing to look for `proto=`.